br/pkg/lightning/mydump/parquet_parser.go

```diff
@@ -453,6 +454,15 @@ func setDatumByString(d *types.Datum, v string, meta *parquet.SchemaElement) {
 	if meta.LogicalType != nil && meta.LogicalType.DECIMAL != nil {
 		v = binaryToDecimalStr([]byte(v), int(meta.LogicalType.DECIMAL.Scale))
 	}
+	if meta != nil && *meta.Type == parquet.Type_BYTE_ARRAY && meta.LogicalType == nil {
```

Comment:
noticed there's `Type_FIXED_LEN_BYTE_ARRAY`, not sure if we should also convert it

Reply:
And how can we make sure we won't include other unwanted types by `*meta.Type == parquet.Type_BYTE_ARRAY && meta.LogicalType == nil`? Can you give some reference

Reply:
> And how can we make sure we won't include other unwanted types by *meta.Type == parquet.Type_BYTE_ARRAY && meta.LogicalType == nil? Can you give some reference

It's more likely an expediency. In the manual of `parquet-go`, I didn't see how they handle bit string differently. 

Perhaps I should've read their code instead.

FYI:  https://pkg.go.dev/github.com/xitongsys/parquet-go#section-readme

Reply:
> noticed there's Type_FIXED_LEN_BYTE_ARRAY, not sure if we should also convert it

According to the manual, `FIXED_LEN_BYTE_ARRAY ` is used to represent `DECIMAL` or `INTERVAL`.